### PR TITLE
SOC-462 service is able to mark one named (id) notification as read

### PR DESCRIPTION
**Pull request by rafalkalinski**
In this version service is removing notification marked as read.

### CODE REVIEW

File: service/notifications/src/main/java/com/wikia/notificationsservice/dao/NotificationDao.java [hunk 1]
```diff
@@ -31,6 +31,10 @@ public Long create(Notification notification) throws JsonProcessingException {
     return nextId;
   }
 
+  public Long delete(Long notificationId) throws IOException {
```
Comment (kvas-damian) [inline service/notifications/src/main/java/com/wikia/notificationsservice/dao/NotificationDao.java]: is ```throws IOException``` required here?

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/InboxIntegrationTest.java [hunk 2]
```diff
@@ -208,6 +208,34 @@ public void notificationGetWithOffsetAndLimit() {
...
+        .userId(new Date().getTime())
+        .build();
+
+    UriBuilder requestUrl = notificationInboxUrl.clone().path(notification.getUserId().toString());
+
+    Response response = client.target(requestUrl)
+        .request()
+        .post(Entity.entity(notification, MediaType.APPLICATION_JSON));
+
+    assertThat(response.getStatus()).isEqualTo(Response.Status.CREATED.getStatusCode());
```
Comment (kvas-damian) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/InboxIntegrationTest.java]: Verifying response status is covered by other test. 
For better readability I suggest limit duplicated asserts to make it easier to follow test steps. What do you think?

Comment (rafalkalinski) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/InboxIntegrationTest.java]: I will remove checking this response status.

**Comment by kvas-damian** [comment]
LGTM

**Comment by pchojnacki** [comment]
LGTM

**Pull request merged**